functions.sh

```diff
@@ -2134,7 +2134,11 @@ format_partitions() {
         # then write swap information
         mkswap "$DEV" 2>&1 | debugoutput ; EXITCODE=$?
       elif [ "$FS" = "ext2" ] || [ "$FS" = "ext3" ] || [ "$FS" = "ext4" ]; then
-        mkfs -t "$FS" -q "$DEV" 2>&1 | debugoutput ; EXITCODE=$?
+        if [ "$IAM" == "centos" ] && [ "$IMG_VERSION" -ge 70 ]; then
```

Comment:
shouldn't the logic be reverted? We want to disable the 64bit features only for centos6, but keep it for everything else?

Reply:
you are right, that logic was incorrect.